src/renderer/widgets/WalletDetails/model/wallet-provider-model.ts

```diff
@@ -29,6 +37,36 @@ const $singleShardAccount = combine(
   },
 );
 
+const $multiShardAccounts = combine(
+  {
+    accounts: $accounts,
+  },
+  ({ accounts }): Map<BaseAccount, Record<ChainId, ChainAccount[]>> => {
```

Comment:
Maybe better create type for `Map<BaseAccount, Record<ChainId, ChainAccount[]>>`?